dev/com.ibm.ws.microprofile.faulttolerance/src/com/ibm/ws/microprofile/faulttolerance/impl/async/QueuedFuture.java

```diff
@@ -52,27 +51,23 @@
      * <p>
      * Don't use it to get the result, use {@link #resultFuture} instead.
      */
-    private Future<Void> taskFuture;
+    private Future<?> taskFuture;
 
     //threadContext is the context retrieved from the originating thread
     private ThreadContextDescriptor threadContext;
     //executionContext is the overall FT execution context, covering both synchronous halves of the asynchronous execution
```

Comment:
you removed the executionContext but not the comment?

Reply:
doh

Reply:
Fixed